### ISSUE TITLE
New versions do not work on LOS 18.1

### ISSUE THREAD

**Issue by xsxrnxmx** [leohearts/AlternativeUnlockXposed#10]
Hello! New versions do not work on Android 11 (I assume the problem is that you changed the directory for storing the PIN code). The video that I attached shows what is happening - the values ​​​​are not saved in any fields, and accordingly the functionality does not work. It will be great if you can come up with something with this :) P.S: the innovation that hides your real pin behind ****** in the module itself is, in my opinion, a rather controversial idea. If an attacker gained physical access to your phone and discovered this module, we can say that you are already in trouble, it doesn’t matter whether you hide your PIN code for the asterisk or not, and it doesn’t matter where the files with the code are stored. The main potential of this application is the ability to unlock subtle changes that you need in the system, hiding what should not be seen by others. A more useful feature would be to add a field for entering a command that should be triggered when entering a real PIN code. Thank you for your attention. I would be very grateful if you can solve the problem with new versions not working and add another field for entering a command :)

https://github.com/user-attachments/assets/d28f7075-a3ee-4040-a43a-88c4fd66a8b0

**Comment by xsxrnxmx** [comment]
There are no errors or problems in LogCat, there is simply nothing to attach.

**Comment by leohearts** [comment]
> There are no errors or problems in LogCat, there is simply nothing to attach.

I can't know what's going wrong without a log. You could search for `avc` , `Posed` and `AlternativeUnlockHook` in your log and try debugging yourself :3

Does the old version work ?

**Comment by xsxrnxmx** [comment]
Yes, the old version (1.3) works. There is nothing in the logs that would indicate an error. And nothing with these keys that you specified (avc, Posed, AlternativeUnlockHook)

**Comment by xsxrnxmx** [comment]
![Screenshot_20241115-141845_LogFox](https://github.com/user-attachments/assets/a56d14dc-2807-430f-964c-d970f1fce930)

**Comment by xsxrnxmx** [comment]
Единственные строки в LogCat, относящиеся к работе модуля.
